tests/specs/lemmas.k

```diff
@@ -110,6 +114,8 @@ module LEMMAS-JAVA [symbolic, kast]
 
     rule #asWord(BUF) /Int 26959946667150639794667015087019630673637144422540572481103610249216 => #asWord(#take(4, BUF)) [simplification]
 
+    rule #asWord(BUF1 : BUF2) >>Int LSHIFT => BUF1                      requires #sizeByteArray(BUF2) *Int 8 ==Int LSHIFT [simplification]
+
```

Comment:
I'm guessing that this is just added for completeness on the Java backend? Because I htoughtt the ERC20 verifications only used the Haskell backend.

Reply:
To make the test pass on Java backend.